app/src/main/java/com/sci/recipeandroid/RecipeApps.kt

```diff
@@ -23,7 +26,11 @@ class RecipeApps: Application() {
                 repositoryModule,
```

Comment:
koin modules start to bloat , plz try to separate functions to group module . eg
cartModules() , authModules() , paymentModules() etc , u cam reference how to do it to separate in Koin doc